public/request/auth/register.php

```diff
@@ -12,6 +12,10 @@
 $pass = $input['password'];
 $email = $input['email'];
 
+if (!isValidUsername($username)) {
```

Comment:
This seemed redundant at first but I just learned that Laravel's `alpha_num` validation rule doesn't prevent usage of "number-like"s.

I'll try to find a better solution for username validation in a separate PR.

Reply:
It also doesn't prevent non-ASCII letters, which was what originally prompted this PR.